helper-scripts/azure-prep-image.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash -e
+#
+# Create an image from a page blob and copy it to a custom region (so as to
+# cirumvent regional CPU limitationf when running tests)
+#
+
+# dummy "az" for testing
+#function az() {
+#	echo "EXEC: az $@"
+#}
+
+function channel_to_region() {
+	local blob_name="$1"
+	local channel=$(echo "$blob_name" \
+						| sed 's/.*\(alpha\|beta\|stable\|edge\).*/\1/')
+
+	case "$channel" in
+		alpha)	echo "uksouth";;
+		beta)	echo "germanywestcentral";;
+		stable) echo "francecentral";;
+		edge)	echo "ukwest";;
+		*)		echo "ERROR: unknown channel '$channel'" >&2
+				exit 1;;
+	esac
+}
+# --
+
+function disk_from_page_blob() {
+	local blob_name="$1"
+	local name="$2"
+	local rg="$3"
+
+	local out=$(mktemp)
+	trap "rm -f $out" EXIT
+
+	az disk create --name "$name" -g "$rg" \
+		--source https://$rg.blob.core.windows.net/publish/"$blob_name".vhd \
+		> "$out"
+	
+	# print disk id
+	cat "$out" | jq -r '.id'
+
+	rm -f "$out"
+	trap - EXIT
+}
+# --
+
+function image_from_disk() {
+	local name="$1"
+	local rg="$2"
+	local disk_id="$3"
+
+	az image create -g $rg --name "$name" \
+						   --source "$disk_id" --os-type linux
+}
+# --
+
+function copy_to_region() {
+	local name="$1"
+	local rg="$2"
+	local dest_rg="$3"
+	local dest_region="$4"
+
+	az image copy --source-resource-group $rg \
```

Comment:
Where is that defined? In the Docker image I don't see it `docker run --rm -it mcr.microsoft.com/azure-cli` (cli version 2.2.0):
```
bash-5.0# az image copy --help
az image: 'copy' is not in the 'az image' command group. See 'az image --help'. If the command is from an extension, please make sure the corresponding extension is installed. To learn more about extensions, please visit https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-overview
bash-5.0# az image copy 
az image: 'copy' is not in the 'az image' command group. See 'az image --help'. If the command is from an extension, please make sure the corresponding extension is installed. To learn more about extensions, please visit https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-overview
bash-5.0# az image --help

Group
    az image : Manage custom virtual machine images.

Subgroups:
    builder [Preview] : Manage and build image builder templates.

Commands:
    create            : Create a custom Virtual Machine Image from managed disks or snapshots.
    delete            : Deletes an Image.
    list              : List custom VM images.
    show              : Gets an image.
    update            : Update custom VM images.
```

Reply:
Same when installed through the installer script.

Reply:
Ah, found it: https://docs.microsoft.com/en-us/cli/azure/ext/image-copy-extension/image?view=azure-cli-latest